app/models/callout.rb

```diff
@@ -63,4 +72,28 @@ class Callout < ApplicationRecord
   def title
     metadata["title"] || "Calout #{id}"
   end
+
+  private
+
+  def validate_commune_ids
+    if commune_ids.blank? || commune_ids.reject!(&:empty?).blank?
```

Comment:
We should have `before_validation` to clean up empty strings, then the validation will be simpler.

```rb
before_validation :clean_up_blank_commune_ids

def clean_up_empty_commune_ids
  self.commune_ids = Array(commune_ids).reject(&:blank?)
end
```